wct.conf.json

```diff
@@ -10,27 +10,17 @@
 			"browsers": [
 				{
 					"browserName": "chrome",
-					"platform": "OS X 10.13",
-					"version": ""
-				},
-				{
-					"browserName": "chrome",
-					"platform": "Windows 10",
+					"platform": "OS X 10.15",
```

Comment:
Slight tweak here as we typically don't bother with Chrome & Firefox testing on both Windows and Mac.